content/01.main_content.md

```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in Le et al. [@doi:10.1016/j.cels.2021.07.007], in order to identify disparities in ISCB honorees, they compared against scientists' publication records.
+  While seemingly straightforward, there exist many choices on how to calculate publication rates. 
+  The authors had to decide which journals would be included in their publication record, whether the publications should be weighted by citation count, and if authorship position should be considered.
+  Choosing the correct reference distribution will be specific to the application, but the selection criteria of the background distribution should be annotated and justified.
+
 Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
-  1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
-  This enables public scrutiny to those being audited and those whose data are used.
-  For confidential data, provide de-identified or aggregated data.
+
+  1. *Transparency.* Publicly provide all results, tools, code, and data used in the analysis.
+  This provides multiple benefits.
+  Firstly, this gives data access to those whose data was used in the analysis.
```

Comment:
What do you think of "Publicly provide all results, tools, code, and data used in the analysis. This gives those whose data was used in the analysis, as well as the general public, the chance to scrutinize the methods used and be informed of any biases found by the audit. For confidential data, de-identified or aggregated data should be provided." ?

I think I asked you to make it wordier and then I changed my mind. :upside_down_face:

Reply:
yeah, i think that sounds better 👍🏼 
added in https://github.com/greenelab/disparities_opinion/pull/16/commits/22d18f281e53af29882742a2cc20e239cc0bb4c2